source/features/clean-issue-filters.tsx

```diff
@@ -59,6 +59,11 @@ async function hideProjects(): Promise<void> {
 }
 
 function init(): void {
+	if (!select.exists('#js-issues-toolbar')) {
+		// Repo has no issues, so no toolbar is shown
+		return;
+	}
```

Comment:
We have `waitForDomReady: false` so we should use `elementReady`.

```suggestion
async function init(): Promise<void | false> {
	if (!await elementReady('#js-issues-toolbar')) {
		// Repo has no issues, so no toolbar is shown
		return;
	}
```

This however means we can drop the other 2 `elementReady` from the page